src/shared/components/section-sort/section-sort.component.scss

```diff
@@ -202,3 +202,20 @@
     }
   }
 }
+
+@media (max-width: 430px) {
+    .section{
+        width: 91%;
+        .select-section{
+            width: 100%;
+            flex-direction: column;
+            gap:0;
+            .select-item{
+                width: 100%;
+            }
+        }
+    }
+    ::ng-deep .alert-wrapper{
```

Comment:
Is this the only way working? No other ways?

Reply:
without ::ng-deep doesnt work